src/server/models/functions/index.ts

```diff
@@ -0,0 +1,24 @@
+import { QueryTypes } from 'sequelize'
+
+import { sequelize } from '../../util/sequelize'
+import {
+  dropLinkStatistics,
+  updateLinkStatistics,
+} from './updateLinkStatistics'
+
+/**
+ * Syncs database functions.
+ */
+export async function syncFunctions() {
+  // Removes any previous implementations of link statistics.
+  await sequelize.query(dropLinkStatistics, {
+    type: QueryTypes.RAW,
+  })
```

Comment:
Remove this, otherwise there will be a brief period of time where the function does not exist. just rely on `CREATE OR REPLACE` instead.

Reply:
Alright, I have removed the drop query